server/src/test/java/org/elasticsearch/search/fetch/subphase/FetchSourcePhaseTests.java

```diff
@@ -191,8 +191,7 @@ private HitContext hitExecuteMultiple(
         SearchExecutionContext sec = mock(SearchExecutionContext.class);
         when(sec.isSourceEnabled()).thenReturn(sourceBuilder != null);
         when(fetchContext.getSearchExecutionContext()).thenReturn(sec);
-        MappingLookup mappingLookup = MappingLookup.EMPTY;
-        when(fetchContext.getSearchExecutionContext().getMappingLookup()).thenReturn(mappingLookup);
+        when(fetchContext.getSearchExecutionContext().getMappingLookup()).thenReturn(MappingLookup.EMPTY);
```

Comment:
I think you want to configure the mock `SearchExecutionContext` to return the empty `MappingLookup`